src/gameLogic/Engine.js

```diff
@@ -1,11 +1,20 @@
 import BoardState from "./BoardState"
 import Ticker from "./Ticker"
+import Player from "./Player"
+import Unit from "./Unit"
+import MovementPattern from "./MovementPattern"
+import ArtificialIntelligence from "./ArtificialIntelligence"
 
 class Engine {
 
   constructor() {
-    this.boardState = new BoardState()
     this.ticker = new Ticker(this.tick.bind(this))
+
+    this.human = new Player("rgb(0, 255, 0)", 0.0)
```

Comment:
Playing with color goes against our design guidelines - we were going to work with shapes instead of colors, unless you want to change that guideline? ;D